content/en/platform/corda/1.6/cenm/backup-recommendations.md

```diff
@@ -0,0 +1,52 @@
+---
+aliases:
+- /cenm-backup-recommendations.html
+date: '2024-06-12'
+menu:
+  cenm-1-6:
+    identifier: cenm-1-6-cenm-backup-recommendations
+    parent: cenm-1-6-operations
+    weight: 185
+tags:
+- backup
+title: CENM backup recommendations
+---
+
+# CENM backup recommendations
+
+Various components of the Corda platform read their configuration from the file system, and persist data to a database or into files on disk.
+Given that hardware can fail, operators of IT infrastructure must have a sound backup strategy in place. Whilst blockchain platforms can sometimes recover some lost data from their peers, it is rarely the case that a node can recover its full state in this way because real-world blockchain applications invariably contain private information (e.g., customer account information). Moreover, this private information must remain in sync with the ledger state. As such, we strongly recommend implementing a comprehensive backup strategy.
+
+The following elements of a backup strategy are recommended:
+
+
+### Database replication
+
+When properly configured, database replication prevents data loss from occurring in case the database host fails.
+In general, the higher the number of replicas, and the further away they are deployed in terms of regions and availability zones, the more a setup is resilient to disasters.
+The trade-off is that, ideally, replication should happen synchronously, meaning that a high number of replicas and a considerable network latency will impact the performance of the Corda nodes connecting to the cluster.
```

Comment:
~~the Corda nodes connecting to the cluster.~~

CENM services connecting to their databases.